.github/workflows/build.yml

```
@@ -45,16 +45,21 @@ jobs:
           CLOUDREPO_CREDENTIALS_KEY: ${{ secrets.CLOUDREPO_CREDENTIALS_KEY }}
 
       - name: Decrypt Git SSH credentials
-        run: ./config/scripts/decrypt.sh "GIT_CREDENTIALS_KEY" ./deploy_key_rsa.gpg ./deploy_key_rsa
+        run: ./config/scripts/decrypt.sh "$GIT_CREDENTIALS_KEY" ./deploy_key_rsa.gpg ./deploy_key_rsa
         env:
           GIT_CREDENTIALS_KEY: ${{ secrets.GIT_CREDENTIALS_KEY }}
 
+        # Make sure the SSH key is not "too visible". SSH agent will not accept it otherwise.
+      - run: chmod 400 ./deploy_key_rsa
+
       - name: Decrypt GCS credentials
-        run: ./config/scripts/decrypt.sh "GCS_CREDENTIALS_KEY" ./gcs-auth-key.json.gpg ./gcs-auth-key.json
+        run: ./config/scripts/decrypt.sh "$GCS_CREDENTIALS_KEY" ./gcs-auth-key.json.gpg ./gcs-auth-key.json
         env:
           GCS_CREDENTIALS_KEY: ${{ secrets.GCS_CREDENTIALS_KEY }}
 
       - name: Publish artifacts to Maven
-        run: ./gradlew assemble publish --stacktrace
+        run: chmod +x ./config/scripts/register-ssh-key.sh & ./gradlew assemble publish --stacktrace
```

`&` makes the commands run simultaneously. `&&` executes the first one and then the second one. Even though `chmod` will always run faster, logically, I believe the intention was to show that the `gradle publish` requires an executable flag on the first script.
```suggestion
        run: chmod +x ./config/scripts/register-ssh-key.sh && ./gradlew assemble publish --stacktrace
```